docs/devel/hotswap.md

```diff
@@ -22,7 +22,7 @@ Adjust `manifests-dev/deployment-dev.yml` to your docker registry image manually
 ## [OPTIONAL] Re-Generate the `manifests-dev/deployment-dev.yml`
 
 You can use `scripts/generate-deployment-dev` to automatically re-generate the
-`manifests-dev/deployment-dev.yml` from `manifests/deployment.yml`.
+`manifests-dev/deployment-dev.yml` from `install/charts/templates/deployment.yml`.
```

Comment:
Does the generation fo the `deployment-dev.yml` from a chart template work?

Reply:
@SAPJonasZengerle

Reply:
There is the possibility to render only the templates locally using ```helm template```. This might be useful.